packages/chat/src/views/ChatWidget/styles.css.ts

```diff
@@ -6,31 +6,34 @@ const MAX_CHAT_HEIGHT = 800;
 
 export const LAUNCHER_MARGIN = 16;
 
+const WIDGET_Z_INDEX = 10000;
+
 export const widgetContainer = recipe({
   base: {
     position: 'fixed',
     inset: 0,
     pointerEvents: 'none',
-    zIndex: 10000,
+    zIndex: WIDGET_Z_INDEX,
   },
   variants: {
     hidden: {
-      true: {
-        display: 'none',
-      },
+      true: {},
     },
     withChat: {
       true: {},
       false: {},
     },
   },
 });
-
 export const chatContainer = style({
-  width: CHAT_WIDTH,
-  maxHeight: MAX_CHAT_HEIGHT,
-  pointerEvents: 'auto',
-
+  maxHeight: 0,
+  pointerEvents: 'none',
+  opacity: 0,
+  width: 0,
+  transition:
+    'opacity 300ms ease-in-out, width 300ms ease-in-out, max-height 300ms ease-in-out, transform 300ms ease-in-out',
```

Comment:
You can use our `transition` method for this, and if you want to modify the settings you can -  
```
transition: transition(['opacity', 'width', 'max-height', 'transform'], { duration: '300ms', timingFunction: 'ease-in-out' })
```